packages/terra-docs/src/Notice.module.scss

```diff
@@ -0,0 +1,34 @@
+// Themes
+@import './clinical-lowlight-theme/Notice.module';
+@import './orion-fusion-theme/Notice.module';
+
+:local {
+  .deprecation {
+    background-color: var(--terra-docs-notice-deprecation-background-color, #fff9e7);
+    border-color: var(--terra-docs-notice-deprecation-border-color, #ffc413);
+    color: var(--terra-docs-notice-deprecation-color, #1c1f21);
+  }
+
+  .maintenance {
+    background-color: var(--terra-docs-notice-maintenance-background-color, #f6ffe7);
+    border-color: var(--terra-docs-notice-maintenance-border-color, #a5ff13);
+    color: var(--terra-docs-notice-maintenance-color, #1c1f21);
```

Comment:
I made up these colors
![](https://media.giphy.com/media/JWiPeGrr8Ms92/giphy.gif)

Reply:
Seems OK to me. But, if you wanted to go official palette, you could use:
```
background-color: var(--terra-docs-notice-maintenance-background-color, #e9f5e0);
border-color: var(--terra-docs-notice-maintenance-border-color, #a5d784);
```

Reply:
how about lowlight?

Reply:
![Artboard](https://user-images.githubusercontent.com/726351/92820331-752d5700-f38f-11ea-8244-01b0b1122f6c.png)
```
// deprecation
background-color: var(--terra-docs-notice-deprecation-background-color, #223225);
border-color: var(--terra-docs-notice-deprecation-border-color, #406c23);
// maintenance
background-color: var(--terra-docs-notice-maintenance-background-color, #483f31);
border-color: var(--terra-docs-notice-maintenance-border-color, #ffb166);
```

Reply:
![Artboard](https://user-images.githubusercontent.com/726351/92820839-069cc900-f390-11ea-9932-a04bcfcd1cb0.png)
`border-color: var(--terra-docs-notice-deprecation-border-color, #a36529);`

That's a bit more balanced.

Reply:
Thanks! updated with the new colors.

Reply:
Looks like Fusion still has the lime green for maintenance.

Reply:
Good call, i'll get that updated